Makefile

```diff
@@ -9,22 +9,23 @@ GOBIN?=$(GOPATH)/bin
 # Build parameters
 OUT?=./out
 DOCKER_TMP?=$(OUT)/docker_temp/
-DOCKER_TAG?=aeraki/consul2istio:latest
+DOCKER_TAG?=docker.io/tanjunchen/consul2istio:latest
```

Comment:
the tag is wrong

Reply:
Where should I push the image? ghcr.io/aeraki-mesh？

Reply:
Yes, we use github packages for releasing aeraki mesh images.

Reply:
updated.